src/libcore/num/mod.rs

```diff
@@ -96,6 +96,7 @@ pub mod dec2flt;
 pub mod bignum;
 pub mod diy_float;
 
+#[macro_export]
 macro_rules! doc_comment {
```

Comment:
```
[00:04:50] error: This node does not have a stability attribute
[00:04:50]    --> libcore/num/mod.rs:100:1
[00:04:50]     |
[00:04:50] 100 | / macro_rules! doc_comment {
[00:04:50] 101 | |     ($x:expr, $($tt:tt)*) => {
[00:04:50] 102 | |         #[doc = $x]
[00:04:50] 103 | |         $($tt)*
[00:04:50] 104 | |     };
[00:04:50] 105 | | }
[00:04:50]     | |_^
```

Reply:
Note that by adding `#[macro_export]` you've added this macro to libcore's *public* API. If you just want to use the macro internally, the necessary step is to add `#[macro_use]` to its module instead.

If you actually want to make `doc_comment!` a public macro, i'll want to bring in someone from libs team to remark on its viability.